test/controllers/entities_controller_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class EntitiesControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @entity = entities(:one)
+  end
+
+  test 'should get index' do
+    get entities_url
+    assert_response :success
+  end
+
+  test 'should get new' do
+    get new_entity_url
+    assert_response :success
+  end
+
+  test 'should create entity' do
+    assert_difference('Entity.count') do
+      post entities_url,
+           params: { entity: { amount: @entity.amount, group_id: @entity.group_id, name: @entity.name,
+                               user_id: @entity.user_id } }
+    end
+
+    assert_redirected_to entity_url(Entity.last)
+  end
+
+  test 'should show entity' do
+    get entity_url(@entity)
+    assert_response :success
+  end
+
+  test 'should get edit' do
+    get edit_entity_url(@entity)
+    assert_response :success
+  end
+
+  test 'should update entity' do
+    patch entity_url(@entity),
+          params: { entity: { amount: @entity.amount, group_id: @entity.group_id, name: @entity.name,
+                              user_id: @entity.user_id } }
+    assert_redirected_to entity_url(@entity)
+  end
+
+  test 'should destroy entity' do
+    assert_difference('Entity.count', -1) do
+      delete entity_url(@entity)
+    end
+
+    assert_redirected_to entities_url
+  end
+end
```

Comment:
- [ ] Optional suggetions 

You have used `minitest` for testing cases, however, you would have used `rspec`, kindly provide clear information in the readme file related to how to run the tests.